msb/broker/config.py

```diff
@@ -0,0 +1,8 @@
+from msb.zmq_base.config import ZMQConf
+
+
+class BrokerConf(ZMQConf):
```

Comment:
Is this the proper way to do it?
An alternative would be creating an alias: 
```python
BrokerConf = ZMQConf
```

Reply:
I like it better with the alias, fixed it.